lib/top/compat_top.ml

```diff
@@ -0,0 +1,285 @@
+open Mdx.Migrate_ast
+
+#if OCAML_MAJOR >= 4 && OCAML_MINOR >= 8
```

Comment:
Cppo supports tuples.
So you can define `#define VERSION (OCAML_MAJOR, OCAML_MINOR)` and compare `#if VERSION >= (4, 8)`.

Reply:
Nice! I didn't know that. I'll submit a PR to change that once this one is merged.